extensions/info/deployment/src/main/resources/dev-ui/qwc-info.js

```diff
@@ -194,7 +194,7 @@ export class QwcInfo extends LitElement {
                     for (const property of Object.keys(extInfo)){
                         rows.push(html`<tr><td class="row-header">${property}</td><td>${extInfo[property]}</td></tr>`);
                     }
-                    cards.push(html`<qui-card header=${key}>
+                    cards.push(html`<qui-card header=${key.charAt(0).toUpperCase() + key.slice(1)}>
```

Comment:
I'm not a big fan of doing this because `gRPC` for instance might be an issue.

Reply:
The others are hard coded into the Card. This section is for looping over all the "custom" injected Info.

Which is set here:
```java
public class FacesInfoContributor implements InfoContributor {

    @Override
    public String name() {
        return "Faces";
    }
```

But if I upper this then the JSON file gets this upper name key.

Reply:
Maybe we should add an Optional<String> displayName field ?

Reply:
But wouldn't that pollute the JSON in the `/q/info` response?

Reply:
@melloware if you don't want it to appear, you can annotate the property with `@JsonIgnore`? That being said, I don't think it will hurt to have the value if defined. It could be useful if you implement a frontend for `/q/info`.

Reply:
OK is it OK if i work on this? adding this new Display Field?